client/src/main/java/org/spine3/change/MessageMismatch.java

```diff
@@ -105,6 +107,10 @@ public static ValueMismatch unexpectedValue(Message expected, Message actual, Me
      * Creates a new instance of {@code ValueMismatch} with the passed values.
      */
     private static ValueMismatch of(Message expected, Message actual, Message newValue, int version) {
+        checkNotNull(expected);
```

Comment:
Why do we do that for a `private static` method?

I can see that's a common issue across the code changes. Please address.